lesson5/Library/renting-event.ts

```diff
@@ -20,10 +20,10 @@ class RentingEvent {
         if (member instanceof Guest) {
             this.expiredDate = this.rentedDate.plus({ day: 10 })
         }
-        if (member instanceof PermanentMember && (member as PermanentMember).isVIP == false) {
+        else if (member instanceof PermanentMember && (member as PermanentMember).isVIP == false) {
             this.expiredDate = this.rentedDate.plus({ day: 20 })
         }
-        if (member instanceof PermanentMember && (member as PermanentMember).isVIP == true) {
+        else {
```

Comment:
Using `else` if & `else` here to tell machine that `this.expiredDate` will never get `undefined`